repairnator/repairnator-dockerpool/src/main/java/fr/inria/spirals/repairnator/dockerpool/RunnablePipelineContainer.java

```diff
@@ -48,6 +49,31 @@ public RunnablePipelineContainer(AbstractPoolManager poolManager, String imageId
         this.treatedBuildTracking = treatedBuildTracking;
         this.skipDelete = skipDelete;
         this.createOutputDir = createOutputDir;
+
+        this.containerName = "repairnator-pipeline_"+ Utils.formatFilenameDate(new Date())+"_"+this.buildId;
+        this.output = (createOutputDir) ? "/var/log/"+this.repairnatorConfig.getRunId() : "/var/log";
+
+        this.envValues = new ArrayList<>();
+        this.envValues.add("BUILD_ID="+this.buildId);
+        this.envValues.add("LOG_FILENAME="+containerName);
+        this.envValues.add("GITHUB_LOGIN="+System.getenv("GITHUB_LOGIN"));
+        this.envValues.add("GITHUB_OAUTH="+System.getenv("GITHUB_OAUTH"));
+        this.envValues.add("RUN_ID="+this.repairnatorConfig.getRunId());
+        this.envValues.add("GOOGLE_ACCESS_TOKEN="+this.repairnatorConfig.getGoogleAccessToken());
+        this.envValues.add("REPAIR_MODE="+this.repairnatorConfig.getLauncherMode().name().toLowerCase());
+        this.envValues.add("SPREADSHEET_ID="+this.repairnatorConfig.getSpreadsheetId());
+        this.envValues.add("PUSH_URL="+this.repairnatorConfig.getPushRemoteRepo());
+        this.envValues.add("MONGODB_HOST="+this.repairnatorConfig.getMongodbHost());
+        this.envValues.add("MONGODB_NAME="+this.repairnatorConfig.getMongodbName());
+        this.envValues.add("SMTP_SERVER="+this.repairnatorConfig.getSmtpServer());
+        this.envValues.add("NOTIFY_TO="+ StringUtils.join(this.repairnatorConfig.getNotifyTo(),','));
+        this.envValues.add("OUTPUT="+output);
+    }
+
+    public RunnablePipelineContainer(AbstractPoolManager poolManager, String imageId, int buildId, int nextBuildId, String logDirectory, TreatedBuildTracking treatedBuildTracking, boolean skipDelete, boolean createOutputDir) {
```

Comment:
Here again, we should use an `InputBuildId`

Reply:
Done.